vizro-core/src/vizro/actions/_callback_mapping/_callback_mapping_utils.py

```diff
@@ -104,13 +104,16 @@ def _get_inputs_of_chart_interactions(
         page=_get_triggered_page(action_id=action_id),
         action_function=action_function,
     )
-    return [
-        State(
-            component_id=_get_triggered_model(action_id=ModelID(str(action.id))).id,
-            component_property="clickData",
+    inputs = []
+    for action in chart_interactions_on_page:
```

Comment:
I think there's quite a bit of renaming we should do since lots of stuff refers explicitly to `graph` or `chart` or the like.

Not sure what the right word is for "component that is reactive to filters and parameters" - maybe just `component` if that's not too ambiguous.

Reply:
I largely agree with this but not sure about the use of the word `chart = component with data_frame = component that responds to filters = graph + table`. What you say is consistent with @maxschulz-COL's view [here](https://github.com/mckinsey/vizro/pull/114#discussion_r1365336478) and is growing on me as an idea.

The reason for my hesitation is that I'm not sure why we shouldn't have all `components` reactive to filters. Like in my demo here, what happened if at some point wanted to make `Card` reactive to filters. Would it become a sort of `chart`? Maybe that would make sense if it is thought of as a chart if it behaves like that and depends on a dataframe. Or maybe this doesn't make sense as an idea so is irrelevant anyway...

Reply:
FYI from a model point of view the difference between `Card` compared to `Table`/`Graph` should I think be that the latter expose a `__call__` method which delegates to a field that has a `CapturedCallable`. So the terminology you're suggesting is basically that `chart` is something that is a callable component, which is probably ok.

Reply:
Let's discuss this as well in our meeting on Monday

Reply:
For completeness, adding:

For now then we say that Figure = something that reacts to filters/parameters = model with figure field. In future we can extend this to further components if needed.